src/transformers/models/llama/modeling_llama.py

```diff
@@ -492,7 +493,7 @@ def forward(
         # TODO: llama does not have dropout in the config??
         # It is recommended to use dropout with FA according to the docs
         # when training.
-        dropout_rate = 0.0  # if not self.training else self.attn_dropout
+        dropout_rate = self.attention_dropout  # if not self.training else self.attn_dropout
```

Comment:
This should still be conditional on `self.training`

Reply:
Yep I also did not implement the logic, will update soon 🤕